Flagsmith.FlagsmithClient/Cache/IdentityTraitsKey.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace Flagsmith
+{
+    public class IdentityTraitsKey
+    {
+        public string Identity { get; set; }
+        public List<ITrait> Traits { get; set; }
+
+        public IdentityTraitsKey(string identity, List<ITrait> traits)
+        {
+            Identity = identity;
+            Traits = traits;
+        }
+
+        public string GenerateUniqueKey()
```

Comment:
I wonder if this could just be included on a wrapper class for the Identity itself somehow?

Reply:
This is kind of the idea for the `IdentityTraitsKey` class that we are in. It is used to identify a unique combinaison of the identity + the specific traits. Or am I not understanding your question?

Reply:
Who will be using this wrapper? Not sure to understand, sorry :)

Reply:
Only our internal code right now would use this, but it seems more extensible to me than creating a separate class solely for the purpose of holding a unique key?

Reply:
all right, got you! Will try to come up with something better today! :)